src/components/Marketplace/MarketplaceScreen.css

```diff
@@ -0,0 +1,18 @@
+
+.searchBarContainer {
```

Comment:
Since this is not fully implemented yet, let's just remove the search bars and tabs so we can make sure your code can get integrated into our codebase. So let's remove all this css